src/actions.js

```diff
@@ -114,14 +114,15 @@ export function receiveBootstrapErrorAction(error) {
   return { type: RECEIVE_BOOTSTRAP_ERROR, error };
 }
 
-export function fetchBootstrap(dispatch, sessionState) {
+export function fetchBootstrap(dispatch, sessionState, dashboardName) {
   if (sessionState && (sessionState.isFetching || sessionState.hasBootstrap)) {
     return;
   }
 
   dispatch(fetchBootstrapAction());
 
-  getBootstrap("dashboard").then(
+  const finalDashboardName = dashboardName ? dashboardName : "dashboard";
```

Comment:
`dashboardName || "dashboard";` is idiomatisch :-)

Reply:
zal ik doen